### ISSUE TITLE
Problems while installing vitepress-plugin-inline-link-preview, cant get ./markdown-it and ./client

### ISSUE THREAD

**Issue by Glomzzz** [nolebase/integrations#159]
I was intalling it in this way: https://nolebase-integrations.ayaka.io/pages/en/integrations/vitepress-plugin-inline-link-preview/getting-started.html

but i couldn't get those:
```
import { 
  InlineLinkPreviewElementTransform 
} from '@nolebase/vitepress-plugin-inline-link-preview/markdown-it'
```
and
```
import { 
  NolebaseInlineLinkPreviewPlugin, 
} from '@nolebase/vitepress-plugin-inline-link-preview/client'
```

well, i'm new to vitepress, i dont  know what to do .

is it a bug? or something else.

**Comment by nekomeowww** [comment]
Oh, this was unreleased yet (for v2), but will be soon released today.

For now, please refer to https://github.com/nolebase/integrations/issues/63 and use

```typescript
import { ElementTransform } from '@nolebase/markdown-it-element-transform'

export default defineConfig({
  markdown: {
    config(md) => {
      md.use(ElementTransform, (() => { 
        let transformNextLinkCloseToken = false 
    
        return { 
          transform(token) { 
            switch (token.type) { 
              case 'link_open': 
                if (token.attrGet('class') !== 'header-anchor') { 
                  token.tag = 'VPNolebaseInlineLinkPreview' 
                  transformNextLinkCloseToken = true 
                } 
                break 
              case 'link_close': 
                if (transformNextLinkCloseToken) { 
                  token.tag = 'VPNolebaseInlineLinkPreview' 
                  transformNextLinkCloseToken = false 
                } 
    
                break 
            } 
          }, 
        } as ElementTransformOptions 
      })())
    }
  }
})
```

with 

```typescript
import { 
  NolebaseInlineLinkPreviewPlugin, 
} from '@nolebase/vitepress-plugin-inline-link-preview'
```

directly.

**Comment by Glomzzz** [comment]
thanks for your braaavo plugins

**Comment by nekomeowww** [comment]
Released as `2.0.0-rc2`, feel free to give it a try 🤭
